document/chapters/03-sdf_preconditioner.tex

```diff
@@ -229,10 +228,8 @@ \subsection{Construction}
 which mesh voxel the point lies within. The point's element is accessed by
 determining an $<i,j,k>$ index using the point's x, y, and z values divided by
 the structured mesh step size. A tri-linear interpolation of the mesh element's 8
-vertex coordinates and their signed distance values is then used to provide the
-signed distance value for the location of interest. As a result, the complexity
-of a signed distance value look-up from the signed distance field is an
-$O(1)$ operation.
```

Comment:
why remove this O(1) declaration?

Reply:
I mention it pretty close to hear already so it felt repetitive, but maybe this was helpful in clarifying where the O(1) component comes from?

Reply:
:+1:  I am only looking at the changes in isolation